### PR TITLE
Bump gem version to 5.5.2

### PR DESCRIPTION
**Pull request by fmlharrison**
Bump bump.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 5.5.2
+
+* Add `full-width` flag to remove left and right padding when using a full width page header.
```
Comment (vanitabarrett) [inline CHANGELOG.md]: Can you put the PR info for that change here too?

Comment (fmlharrison) [reply CHANGELOG.md]: Damn! Cheers for picking that up!

**Comment by vanitabarrett** [comment]
I think we can probably include https://github.com/alphagov/govuk_publishing_components/pull/202 in this version bump too. I'm going to merge it now and then we can update the CHANGELOG here.

**Comment by vanitabarrett** [comment]
@fmlharrison Are we able to squash everything into one commit, including the merge commit?

**Comment by leenagupte** [comment]
Replaced with PR #215

**Pull request closed without merging**